packages/nextcloud/lib/nextcloud.dart

```diff
@@ -1,24 +1,17 @@
-library nextcloud;
-
-import 'dart:convert';
-import 'dart:typed_data';
-
-import 'package:crypto/crypto.dart';
-import 'package:nextcloud/nextcloud.dart';
-import 'package:nextcloud/src/nextcloud.openapi.dart' as openapi;
-import 'package:universal_io/io.dart';
-import 'package:version/version.dart';
-import 'package:xml/xml.dart' as xml;
-
 export 'package:crypton/crypton.dart' show RSAKeypair, RSAPrivateKey, RSAPublicKey;
 
-export 'src/nextcloud.openapi.dart' hide NextcloudClient;
+export 'src/api/core.openapi.dart' hide jsonSerializers, serializers;
+export 'src/api/news.openapi.dart' hide jsonSerializers, serializers;
+export 'src/api/notes.openapi.dart' hide jsonSerializers, serializers;
+export 'src/api/notifications.openapi.dart' hide jsonSerializers, serializers;
+export 'src/api/provisioning_api.openapi.dart' hide jsonSerializers, serializers;
+export 'src/api/uppush.openapi.dart' hide jsonSerializers, serializers;
+export 'src/api/user_status.openapi.dart' hide jsonSerializers, serializers;
```

Comment:
we should annotate the hidden fields with @internal so we don't accidentally expose them in the future.

You probably need to add the `meta` package to nextcloud first.